docs/tasks/manage-stateful-set/debugging-a-statefulset.md

```diff
@@ -77,7 +77,9 @@ kubectl annotate pods <pod-name> pod.alpha.kubernetes.io/initialized="true" --ov
 {% endcapture %}
 
 {% capture whatsnext %}
-Learn more about debugging an init-container. *TODO: Link to Task: Debugging an init-container*
+
+Learn more about [debugging an init-container](/docs/tasks/troubleshoot/debug-init-containers/).
```

Comment:
GitHub won't let me comment up there, but the "Deleting StatefulSet Pods" link is broken. It should be `/docs/tasks/manage-stateful-set/delete-pods/`.